README.md

```diff
@@ -277,7 +277,15 @@ Two steps to set up debugging:
    For newer OCaml version support, opam pin the development version from
    <https://github.com/hackwaly/ocamlearlybird>.
 
-2. Build _bytecode_ version of your OCaml program executable.
+2. If you use `dune` 3.0+ to build your project, switch to 3.7+ and make sure that you add:
```

Comment:
It would be more precise to refer to the `(lang dune ...)` used by the project, rather than the version of dune itself. For example, `(lang dune 2.8)` still works fine even if dune 3.7 is installed.

Reply:
@sim642 thanks, changed!